lib/formatters/tapFormatter.mjs

```diff
@@ -37,7 +39,7 @@ export default function tapFormatter(results) {
 
 				for (const { text, severity, line, column, endLine, endColumn } of warnings) {
 					lines.push(
-						`    - message: "${text}"`,
+						`    - message: ${yaml.dump(`${text}`, { quotingType: '"', forceQuotes: true, lineWidth: -1 }).trim()}`,
```

Comment:
note

`lineWidth` added to avoid the strip block chomping indicator (default is 80)